src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -80,6 +85,106 @@ enum MiscSpells
     SPELL_MAGE_CHILLED                           = 205708
 };
 
+// 110909 - Alter Time Aura
+// 342246 - Alter Time Aura
+class spell_mage_alter_time_aura : public AuraScript
+{
+    PrepareAuraScript(spell_mage_alter_time_aura);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo
+        ({
+            SPELL_MAGE_ALTER_TIME_VISUAL,
+            SPELL_MAGE_MASTER_OF_TIME,
+            SPELL_MAGE_BLINK,
+        });
+    }
+
+    void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+    {
+        Unit* unit = GetTarget();
+        if (unit == nullptr)
+            return;
+
+        if (Player* player = unit->ToPlayer())
```

Comment:
`GetHealth()` and `GetWorldLocation()` are not specific to `Player` class, those are `Unit` and `WorldLocation` member functions (you don't need to use `ToPlayer()` here)